contracts/multisig/src/multisig_sign.rs

```diff
@@ -73,6 +92,11 @@ pub trait MultisigSignModule:
         let (caller_id, caller_role) = self.get_caller_id_and_role();
         require!(caller_role.can_sign(), "only board members can un-sign");
 
+        let group_status = self.action_group_status(group_id).get();
+        require!(
+            group_status == ActionStatus::Available,
+            "cannot sign actions of an aborted batch"
```

Comment:
Wrong error message here.
+ now I think because of this, users cannot discard a batch if it was aborted, thus remove it from state, right?
I can look further but a unit test will be better here